infrastructure/src/main/java/org/corfudb/infrastructure/SequencerServer.java

```diff
@@ -85,6 +84,9 @@
 @Slf4j
 public class SequencerServer extends AbstractServer {
 
+    // Default cache size for transaction conflict resolution
+    private static final long DEFAULT_CACHE_SIZE = 250_000;
```

Comment:
Done. Actually this default value is already exposed in CorfuServer configurations. Removed this variable since it's redundant. Moved get config logic to ServerContext.